src/api/spec/controllers/webui/package_controller_spec.rb

```diff
@@ -589,7 +589,7 @@ def remove_file_post
       login(user)
     end
 
-    context 'when triggering a rebuild fails' do
+    context 'non existant repository' do
```

Comment:
```suggestion
    context 'non existent repository' do
```

Reply:
this is misspelled in other places of this context as well, maybe you can correct those as well :)

Reply:
That's not the purpose of this pull request, sorry. That can be addressed in a following pull request.

Reply:
I applied your suggestion. This misspelling was introduced in this pull request. Please take a look again.

The rest of the misspellings could be addressed later on in other pull requests.